{"name": "Order creation without ingredients but with auth error test", "status": "passed", "start": 1732715199367, "stop": 1732715201467, "uuid": "e27862ef-0cf4-49bd-9732-5efb04327887", "historyId": "f1566bedc63da0966226b8a581690569", "testCaseId": "f1566bedc63da0966226b8a581690569", "fullName": "tests.test_order_creation.TestCreateOrder#test_create_order_no_ingredients_with_auth", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_creation"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "ast24help1c2"}, {"name": "thread", "value": "11252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_creation"}]}